src/components/tell-me-modal/tell-me-modal.ts

```diff
@@ -0,0 +1,45 @@
+import { Component } from '@angular/core';
+import { ViewController, NavParams } from 'ionic-angular';
+
+/**
+ * Generated class for the TellMeModalComponent component.
+ *
+ * See https://angular.io/api/core/Component for more info on Angular
+ * Components.
+ */
+@Component({
+  selector: 'tell-me-modal',
+  templateUrl: 'tell-me-modal.html'
+})
+export class TellMeModalComponent {
+  selectedTellMeQuestionId: string;
+
+  tellMeQuestions = [
+    { id: 'T1', today: 2, last7days: 8 },
```

Comment:
Can we use real question text instead of numbers? We don't want the examiners to go out with their show me/tell me cards?

Reply:
hey Daniel, I implement it according to the designs and I got your point and I have asked the similar thing to Neil. He said that this screen will be revisited and I can leave it like this.